### PR TITLE
SW-2161 Admin UI for importing shapefiles

### PR DESCRIPTION
**Pull request by sgrimm**
Add a section to the admin UI to allow importing shapefiles to create planting
sites, planting zones, and plots. The UI allows various validation checks to be
disabled; this allows us to work with shapefiles that haven't yet been made to
comply exactly with our product requirements.

Currently this is a write-only capability. There isn't yet a way to query
existing planting site data.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #783** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/783" target="_blank">Graphite</a> 
    * **PR #784** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/784" target="_blank">Graphite</a> 
      * **PR #785** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/785" target="_blank">Graphite</a>  👈
        * **PR #787** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/787" target="_blank">Graphite</a> 
          * **PR #788** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/788" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/785?utm_source=stack-comment).

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #783** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/783" target="_blank">Graphite</a> 
    * **PR #784** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/784" target="_blank">Graphite</a> 
      * **PR #785** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/785" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/785?utm_source=stack-comment).

File: src/main/resources/templates/admin/shapefile.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+</style>
+<body>
+
+<span th:include="/admin/header :: top"/>
+
+<a th:href="|${prefix}/|">Home</a>
+
+<h2>Shapefile viewer</h2>
+
+<!-- TODO: Permission for this -->
```
Comment (karthikbtf) [inline src/main/resources/templates/admin/shapefile.html]: does the canCreateFacility gating satisfy this TODO?

Comment (sgrimm) [reply src/main/resources/templates/admin/shapefile.html]: Oops, forgot to delete this. Thanks.

**Pull request merged**